CONTRIBUTING.md

```diff
@@ -85,7 +85,10 @@ SYMFONY_DEPRECATIONS_HELPER=disabled ./bin/phpunit tests/Unit
 ### Fixtures
 
 You might want to load random data to database instead of manually adding magazines, users, posts, comments etc.
-To do so, execute: `bin/console doctrine:fixtures:load --append --no-debug`
+To do so, execute: `php bin/console doctrine:fixtures:load --append --no-debug`
+You should stop the messenger processes, as they are not able to access the data while it is being created, but try to do so.
```

Comment:
Just for my understanding: why aren't they able to access the data?

Reply:
I added that explanation to the issue

Reply:
Thanks, that's clear.